keyboards/handwired/marauder/keymaps/orvia/rules.mk

```
@@ -0,0 +1,29 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = yes            # Link Time Optimization, shrinks the output slightly
+OLED_DRIVER_ENABLE = yes    # OLED Driver Enable
+WPM_ENABLE = yes			# WPM counting Enable
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```

This file should just be overrides, for example
```suggestion
VIA_ENABLE = yes
```

---

That particular keymap is not named via simply because I have experimented with a lot of things there. But I agree the overrides only in the rules.mk, so I've resolved that, thank you.

Added a via keymap for consistency as well, thank you.